cpp/log/cert.h

```diff
@@ -30,6 +30,7 @@ class Cert {
   // Will return null if |x509| is null.
   static std::unique_ptr<Cert> FromX509(ScopedX509 x509);
 
+  Cert() = delete;
```

Comment:
Yeah, I considered just deleting it, but I think it's nice that there's no ambiguity about not being able to construct one (the other constructor is more of an implementation detail).